docs/flyte_agents/developing_agents.md

```diff
@@ -159,29 +159,15 @@ For flytekit versions `>v1.10.2`, use `pyflyte serve agent`.
 
 ### 4. Deploy Your Flyte agent
 
-1. Update the FlyteAgent deployment's [image](https://github.com/flyteorg/flyte/blob/master/charts/flyteagent/templates/agent/deployment.yaml#L35)
+Update the FlyteAgent deployment's [image](https://github.com/flyteorg/flyte/blob/master/charts/flyteagent/templates/agent/deployment.yaml#L35)
 
 ```bash
 kubectl set image deployment/flyteagent flyteagent=ghcr.io/flyteorg/flyteagent:latest
 ```
 
-2. Update the FlytePropeller configmap:
-
-```YAML
- tasks:
-   task-plugins:
-     enabled-plugins:
-       - agent-service
```

Comment:
we still need `enabled-plugins`, right?

Reply:
after this PR is merged, we don't need that.
https://github.com/flyteorg/flyte/pull/5730